include/swift/ABI/MetadataValues.h

```diff
@@ -2028,12 +2028,18 @@ inline int descendingPriorityOrder(JobPriority lhs,
   return (lhs == rhs ? 0 : lhs > rhs ? -1 : 1);
 }
 
+inline JobPriority withPriorityDowngrade(JobPriority priority) {
+  return (priority == JobPriority::UserInteractive) ? JobPriority::UserInitiated
```

Comment:
Why does this sometimes ignore the passed-in priority? Might be worth a brief comment explaining that.

Reply:
I don't think this name clearly identifies what it does; I would say "downgradingUserInteractive` or something like that.